plugins/modules/iosxr_lacp.py

```diff
@@ -251,6 +261,75 @@
 #
 
 
+# Using parsed
+# parsed.cfg
+# ------------
+#
+# lacp system mac 00c1.4c00.bd15
+# lacp system priority 11
+# - name: Convert LACP config to argspec without connecting to the appliance
+#   cisco.iosxr.iosxr_lacp:
+#     running_config: "{{ lookup('file', './parsed.cfg') }}"
```

Comment:
@rohitthakur2590 can u directly pass the cmds here instead of ./parsed.cfg file.

Reply:
@justjais  well it's one of the two ways to provide config for parsing, I have observed in ios-xr most of the parsed examples have file input so I kept this consistent, but yeah in the newer modules we could also do that. IMO for now, this should be fine.

Reply:
@rohitthakur2590 I understand, but as we are not giving content of ./parsed.cfg in the docs, it'd be better if we can pass it directly as otherwise user won't be able to understand what's there in parsed.cfg, else u can copy the content of parsed.cfg in doc examples, even that should be ok as well.

Reply:
@justjais  contents are there right?

Reply:
@rohitthakur2590 Please remove # from parsed play